### DISCUSSION TITLE
Is there any documentation for ColorPicker in 11.0.0-rc1.1?

### DISCUSSION THREAD

**Discussion by bodong1987** [AvaloniaUI/Avalonia#11841] [Q&A] [answered]
I used the latest VisualStudio plug-in to create a desktop program, and then replaced the Content of MainView from TextBlock to ColorPicker, but after modifying various parameters, the ColorPicker could not be displayed. I don’t know if there is anything else to sit on. Did I overlook something?

For the template, the only change is MainView.axml:
```
<UserControl xmlns="https://github.com/avaloniaui"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
             xmlns:vm="clr-namespace:AvaloniaApplication18.ViewModels"
             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
             x:Class="AvaloniaApplication18.Views.MainView"
             xmlns:controls="using:Avalonia.Controls"
             x:DataType="vm:MainViewModel">
  <Design.DataContext>
    <!-- This only sets the DataContext for the previewer in an IDE,
         to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
    <vm:MainViewModel />
  </Design.DataContext>

    <Grid ColumnDefinitions="Auto,*" RowDefinitions="Auto,*">
        <ColorPicker 
            Grid.Row="0"
            Grid.Column="0"            
            HsvColor="hsv(120, 1, 1)"
            Margin="0,50,0,0"
            Width="120"
            Height="50"
            >
            <ColorPicker.Palette>
                <controls:FlatHalfColorPalette />

            </ColorPicker.Palette>
        </ColorPicker>
    </Grid>
</UserControl>
```
![image](https://github.com/AvaloniaUI/Avalonia/assets/9072328/9afabd4d-4ba7-4fa4-94e3-a81bc2ef8206)
[AvaloniaApplication18.zip](https://github.com/AvaloniaUI/Avalonia/files/11795073/AvaloniaApplication18.zip)

**Comment by rabbitism** [comment, marked as answer]
ColorPicker package needs to be installed separately, and styles needs to be included separately in App.axaml.

**Reply by robloo** [reply]
This is correct for the issue you are having. The docs are here: https://github.com/AvaloniaUI/Documentation/blob/04a0b4791e73ede0353f869766042bf4acb64fae/docs/controls/colorpicker/colorcontrols.md#using-in-your-app

**Comment by robloo** [comment]
There is some draft documentation but I'm waiting for when (if) the new docs website goes public before I finish it. The draft is currently in the old repository at the link below. That said, it should be very useful for setting this up right now:

https://github.com/AvaloniaUI/Documentation/blob/04a0b4791e73ede0353f869766042bf4acb64fae/docs/controls/colorpicker/colorcontrols.md

All properties and members of all ColorPicker controls are fully documented in code as well -- that should help too.

**Reply by MikeCodesDotNET** [reply]
New docs are coming along nicely. You can see them [here](https://zealous-desert-09e073403.3.azurestaticapps.net/).

Feel free to submit a PR in [this repo](https://github.com/AvaloniaUI/avalonia-docs).

**Comment by bodong1987** [comment]
thank you...
